TeamCode/src/main/java/org/firstinspires/ftc/teamcode/localization/Localizer.java

```diff
@@ -0,0 +1,25 @@
+package org.firstinspires.ftc.teamcode.localization;
+
+import com.acmerobotics.roadrunner.Pose2d;
+import com.acmerobotics.roadrunner.PoseVelocity2d;
+
+/**
+ * Interface for localization methods.
+ */
+public interface Localizer {
+    void setPose(Pose2d pose);
+
+    /**
+     * Returns the current pose estimate.
+     * NOTE: Does not update the pose estimate;
+     * you must call update() to update the pose estimate.
+     * @return the Localizer's current pose
+     */
+    Pose2d getPose();
```

Comment:
Would getCachedPose or something be better for this, to make it clear that you're not actually reading the pose from hardware?

Reply:
perhaps? `pose` by itself was the name of the property in the robot drive classes so this feels like the logical extension of that

Reply:
I think `getPose()`. (IMO the SDK is straying from convention by using the getter/setter nomenclature for costly I/O operations.)